src/app/onion/learning-object-builder/components/scaffold/hierarchy/hierarchy.component.scss

```diff
@@ -0,0 +1,80 @@
+.children-list {
+    width: 500px;
+    max-width: 100%;
+    min-height: 60px;
+    display: block;
+    background: white;
+    border-radius: 1px;
+    overflow: hidden;
+  }
+  
+  .child-box {
+    padding: 20px 10px;
+    border-bottom: solid 1px#ccc;
```

Comment:
Why is this color hard coded? Can we use one of the defined SCSS variables?

Reply:
Resolved with Add style color